src/_guides/language/sound-faq.md

```diff
@@ -0,0 +1,189 @@
+compilation. AOT compiling benefits significantly from strong type
+checking that can be performed at compile time. Strong mode enables
+the Dart team to build tools that make for a better development experience.
+Also, our developers have told us that they prefer more feedback from
+their type system, to help them manage larger code bases and better
+understand their code.
+
+### How does strong mode benefit me, the developer?
+
+Strong mode helps you find more bugs at compile time, rather than runtime.
+Also, refactoring tools and code completion work better with strong mode.
+
+### How does strong mode benefit my users?
+
+Code that is easier to develop, maintain, and debug is ultimately also
+better for your users. It ensures that you catch more bugs before you ship!
+In future, sound Dart will allow our compilers to generate smaller,
+faster code.
+
+### Yikes, another mode for Dart! So confusing! Why do I need this?
+
+The Dart team is exploring ways of unifying checked mode and
+strong mode to simplify things in the future.
+
+### Is strong mode specified? If so, where is the spec?
+
+The strong mode spec has not yet been published. However, the
+[Sound Dart Guide](/guides/language/sound-dart) has more information
+with links under
+[Other references](/guides/language/sound-dart#other-resources)
+to docs written by the Dart language team.
+
+### Is strong mode "done" or are there still changes to come?
+
+There may be further changes, but the bulk of the work for strong mode is
+done. For example, generic methods have been added to Dart to make it
+easier to write sound code. We try to keep changes to a minimum,
+particularly breaking changes, and to let developers know well in advance.
+For more information on using generic methods now, see
+[Using generic methods](/guides/language/language-tour#using-generic-methods).
+
+### What changed in strong mode from Dart's "classic mode"?
+
+Only a few rules have been added to make Dart a sound language.
+Primarily, you must learn the rules for when you can substitute a
+type with its subtype or supertype. You must use the proper return
+types and input parameter types when overriding methods.
+Finally, you should never “hide” other types in a dynamic collection
+or dynamic instance. See
+[Sound type system](sound-dart-guide#sound-type-system)
+for an explanation of these changes.
+
+### Can I use strong mode today?
+
+Yes! Strong mode is enforced by Dart Analyzer and can be enabled today.
+For more information, see
+[How to enable strong mode](sound-dart-guide#how-to-enable-strong-mode).
+
+### How do I use strong mode today?
+
+For information on using strong mode today, see
+[How to enable strong mode](sound-dart-guide#how-to-enable-strong-mode).
+
+### Does strong mode affect the runtime behavior of my code?
+
+Strong mode hasn’t affected the runtime behavior for the Dart VM or dart2js.
+Strong mode may affect the runtime behavior in DDC if it inserts
```

Comment:
First time DDC is mentioned. This should link to somewhere and also include the full name.